wordpress-mobile/WordPress-iOS
# WordPressAuthenticator: pass the new style parameter for spinner color when NUXButton is disabled

Fixes https://github.com/woocommerce/woocommerce-ios/issues/2716
WordPressAuthenticator PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/562

- [x] ⚠️ Update pod to release version after https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/562 is merged ⚠️ 

## Changes

This PR set the new style parameter `WordPressAuthenticatorStyle.disabledButtonActivityIndicatorColor` to the same value as before (`WordPressAuthenticatorStyle.secondaryTitleColor`).

## To test

- Run `bundle exec pod install --repo-update`
- Launch the app in logged out state, or log out from the app
- Tap "Continue with WordPress.com"
- Enter an email
- Tap "Continue" --> the spinner color should look the same as before
- Go back to the prologue screen
- Tap "Enter your existing site address"
- Enter a site address
- Tap "Continue" --> the spinner color should look the same as before

## Example screenshots

(No changes are expected!)

\ | before | after
-- | -- | --
log in with email | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 17 06 36](https://user-images.githubusercontent.com/1945542/104574088-9cbf6380-5690-11eb-9ece-f45f06c94cd1.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 17 31 07](https://user-images.githubusercontent.com/1945542/104574105-9fba5400-5690-11eb-896a-7acb573cb4d0.png)
log in with site address | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 17 07 07](https://user-images.githubusercontent.com/1945542/104574141-a943bc00-5690-11eb-95f6-ab31e4c2b573.png) | ![Simulator Screen Shot - iPhone 8 Plus - 2021-01-14 at 17 31 18](https://user-images.githubusercontent.com/1945542/104574147-acd74300-5690-11eb-90ec-781c064ed46b.png)

PR submission checklist:
